db/migrate/20151006151101_create_trips.rb

```diff
@@ -1,12 +0,0 @@
-class CreateTrips < ActiveRecord::Migration
```

Comment:
Migrations sollte man lieber nicht löschen, da man dann nicht einfach mit 'rake db:migrate' migrieren kann, sondern erst die gesamste Datenbank droppen und wieder erstellen muss, was sich vor allem in Production ungünstig macht. Erstelle lieber eine Migration, die die Attribute entfernt oder hänge das an deine bestehende create_trips migration an.